internal/static/parse.go

```diff
@@ -0,0 +1,72 @@
+package static
+
+import (
+	"go/constant"
+	"go/types"
+
+	"github.com/dogmatiq/configkit"
+	"github.com/dogmatiq/configkit/internal/entity"
+	"github.com/dogmatiq/configkit/internal/typename/gotypes"
+	"golang.org/x/tools/go/ssa"
+)
+
+// parse parses the member of the package and populates it into
+// entity.Application.
+//
+// The second parameter is the type that actually implements
+// github.com/dogmatiq/dogma.Application interface.
+func parse(m ssa.Member, typ types.Type) *entity.Application {
```

Comment:
```suggestion
func analyzeApplication(m ssa.Member, typ types.Type) *entity.Application {
```

We're not really "parsing" anything here, so I think we should move away from that terminology across the board.

The file would need renaming too. Perhaps it could be `application.go` in anticipation of future `analyzeBlah()` functions for the handlers, or otherwise just be called `analyze.go` for now.